__tests__/browser.ts

```diff
@@ -16,16 +16,34 @@ describe('browser', () => {
     page.on('console', msg => {
       for (let i = 0; i < msg.args().length; ++i) {
         /* eslint-disable-next-line no-console */
-        console.log(`${i}: ${msg.args()[i]}`)
+        console.log(msg.args()[i])
       }
     })
 
+    page.on('pageerror', function(err) {
+      /* eslint-disable-next-line no-console */
+      console.log('Page error: ' + err.toString())
+    })
+
+    page.on('error', function(err) {
+      /* eslint-disable-next-line no-console */
+      console.log('Error: ' + err.toString())
+    })
+
     await page.addScriptTag({
       path: resolve(
         __dirname,
         '../packages/swarm-browser/dist/erebos.swarm.development.js',
       ),
     })
+
+    await page.addScriptTag({
+      path: resolve(
+        __dirname,
+        '../packages/swarm-browser/dist/readable-stream.js',
```

Comment:
I had to get `readable-stream` inside the puppeteer browser somehow. I came up with this, yet if you have some better ideas I am open to them.

Reply:
This is only needed for `puppeteer`. Yes it is bundled using Webpack but the dependency is not exposed for external use and since I have to construct the stream for the test I had to have access to it somehow and this is the solution I came up with...